collect_app/src/test/java/org/odk/collect/android/formmanagement/FormMetadataParserTest.java

```diff
@@ -60,43 +59,17 @@ public void canParseFormWithLastSaved() throws Exception {
         File formXml = File.createTempFile("form", ".xml");
         FileUtils.write(formXml, LAST_SAVED.getBytes());
 
-        FormMetadataParser formMetadataParser = new FormMetadataParser(referenceManager);
+        FormMetadataParser formMetadataParser = new FormMetadataParser();
         Map<String, String> metaData = formMetadataParser.parse(formXml, mediaDir);
         assertThat(metaData.get(FileUtils.FORMID), is("basic-last-saved"));
     }
 
-    /**
```

Comment:
This will now parse with the placeholder instance but there will be no choices to pick from. That case is tested in JavaRosa. It feels esoteric enough that it doesn't need to be tested again.